mmhuman3d/data/datasets/pipelines/transforms.py

```diff
@@ -353,6 +388,35 @@ def __call__(self, results):
             body_pose = smpl_pose_flipped[3:]
             results['smpl_global_orient'] = global_orient
             results['smpl_body_pose'] = body_pose.reshape((-1, 3))
+
+        if 'smplx_body_pose' in results:
+            
+            body_pose = results['smplx_body_pose'].copy().reshape((-1))
+            body_pose_flipped = _flip_smplx_pose(body_pose)
+            results['smplx_body_pose'] = body_pose_flipped
+        
+        if 'smplx_global_orient' in results:
+            global_orient = results['smplx_global_orient'].copy().reshape((-1))
+            global_orient_flipped = _flip_axis_angle(global_orient)
+            results['smplx_global_orient'] = global_orient_flipped
+        
+        if 'smplx_jaw_pose' in results:
+            jaw_pose = results['smplx_jaw_pose'].copy().reshape((-1))
+            jaw_pose_flipped = _flip_axis_angle(jaw_pose)
+            results['smplx_jaw_pose'] = jaw_pose_flipped
+
+        if 'smplx_right_hand_pose' in results:
+            right_hand_pose = results['smplx_right_hand_pose'].copy()
+            left_hand_pose = results['smplx_left_hand_pose'].copy()
+            results['smplx_right_hand_pose'], results['smplx_left_hand_pose'] = _flip_hand_pose(right_hand_pose,left_hand_pose)
+            
+
+
+        # Expressions are not symmetric, so we remove them from the labels when the image is flipped
```

Comment:
Is this the official way?

Reply:
This is the official description from ExPose.